public/views/includes/sidebar.html

```diff
@@ -1,6 +1,6 @@
 <nav class="sidebar left-off-canvas-menu" ng-controller="sidebarController as sidebar">
   <header>
-    <img src="img/logo-negative.svg" alt="Copay" width="80">
+    <logo negative="true" width="80"/>
```

Comment:
The logo element has to be closed by `</logo>`. If it 's not closed, the element below is replaced. I figured it out because the version below logo disappeared.